api/serializers.py

```diff
@@ -198,10 +240,53 @@ class ListEventSerializer(serializers.ModelSerializer):
     countries = MiniCountrySerializer(many=True)
     field_reports = MiniFieldReportSerializer(many=True, read_only=True)
     dtype = DisasterTypeSerializer()
+
+    class Meta:
+        model = Event
+        fields = ('name', 'dtype', 'countries', 'summary', 'num_affected', 'ifrc_severity_level', 'glide',
+                  'disaster_start_date', 'created_at', 'auto_generated', 'appeals', 'is_featured', 'is_featured_region',
+                  'field_reports', 'updated_at', 'id', 'slug', 'parent_event')
+
+class ListEventCsvSerializer(serializers.ModelSerializer):
+    appeals = serializers.SerializerMethodField()
+    field_reports = serializers.SerializerMethodField()
+    countries = serializers.SerializerMethodField()
+    dtype = DisasterTypeSerializer()
+
+    def get_countries(self, obj):
+        country_fields = {}
+        countries = obj.countries.all()
+        if len(countries) > 0:
+            country_fields['id'] = ', '.join([str(country.id) for country in countries])
+            country_fields['name'] = ', '.join([str(country.name) for country in countries])
+        else:
```

Comment:
We should avoid using `len(qs)`. Also instead of fetching all countries we should only fetch required field. Something like this.
```suggestion
        if countries.exists():
            country_fields['id'] = ', '.join([str(id) for id in countries.values_list('id', flat=True)])
            country_fields['name'] = ', '.join(countries.values_list('name', flat=True))
        else:
```

Reply:
Thanks! Commiting this for now, @kamicut tell me if we'd still want to keep your version so I can revert this back if needed.